R/derive_vars_transposed.R

```diff
@@ -14,6 +14,10 @@
 #'
 #'  Keys used to merge `dataset_merge` with `dataset`.
 #'
+#' @param id_vars ID variables
+#'
+#'  Variables (excluding by_vars) that uniquely identify each observation in `dataset_merge`.
```

Comment:
ah i see dataset_merge in the code of the function.  Since the user won't see this I would say something more generic.